ivre/db/mongo.py

```diff
@@ -1127,6 +1127,19 @@ def searchports(self, ports, protocol='tcp', state='open', neg=False):
             for port in ports]}}
 
     @staticmethod
+    def searchobjectid(listId=None, neg=False):
+        assert listId is not None
```

Comment:
Is there a reason I have missed why you do not want to use a mandatory parameter here?

IMO, the method "prototype" should be:
```Python
@staticmethod
def searchobjectid(oid, neg=False):
 ```

Then, you should consider that either a single object ID has been passed, or a list (or tuple, or even a generator, maybe from a MongoDB cursor for example).

By the way, when adding new classes or methods, please make sure to add an appropriate docstring with it.

Since `bson.objectid.ObjectId()` constructor will happily handle an argument that happens to be already an `ObjectId`, it is OK to call it for each element of the list regardless of its type, but the docstring should make it clear that it is OK to use either strings or object IDs.